source/irfs/point_like/index.rst

```diff
@@ -15,49 +15,66 @@ Any point-like IRF component should contain the header keyword:
 
 * ``HDU_CLAS3 = POINT-LIKE``
 
-``RAD_MAX`` column
-------------------
+.. _rad_max:
 
-In addition, binary tables should contain the ``RAD_MAX`` column, containing the radial cut applied
-to calculate the IRF component (unit: deg). As this value is allowed to change as a function of the energy and field 
-of view (FoV) coordinates, the dimension of this column is equal to: 
+``RAD_MAX``
++++++++++++
 
-* ndim: 2 in case that the FoV coordinate is ``THETA``
-* ndim: 3 in case that the FoV coordinates are ``DETX`` and ``DETY``
+In addition to the IRFs, the actual directional cut applied to the data needs to be stored. This cut is allowed 
+to be constant or variable along several axes, with a different format.
 
-As an example, the format of a point-like effective area (as a function of the true energy) is shown below. 
+In case the angular cut is constant along the energy and FoV, an additional header keyword may be added to the 
+IRF HDU:
 
-Example: point-like effective Area vs true energy
--------------------------------------------------
+Header keyword:
 
-Columns:
+* ``RAD_MAX`` type: float, unit: deg
+    * Radius of the directional cut applied to calculate the IRF, in degrees.
+
+If this keyword is present, the science tools should assume the directional cut of this point-like IRF is constant 
+over all axes. In case the angular cut is variable along any axis (reconstructed energy or FoV), an additional HDU 
+is required to store these values. Note any DL3 file with a point-like IRF (with ``HDU_CLAS3`` = POINT-LIKE) that 
+has no ``RAD_MAX`` keyword within the HDU metadata should have this additional HDU.
+ 
+In case the directional cut is variable with energy or the FoV, point-like IRFs require an additional binary 
+table. It stores the values of ``RAD_MAX`` as a function of the reconstructed energy and FoV following the 
+:ref:`fits-arrays-bintable-hdu` format.
+
+.. _rad_max_2d:
+
+``rad_max_2d``
+--------------
+
+The ``rad_max_2d`` format contains a 2-dimensional array of directional cut values, stored in the 
+:ref:`fits-arrays-bintable-hdu` format.
+
+Required columns:
 
 * ``ENERG_LO``, ``ENERG_HI`` -- ndim: 1, unit: TeV
-    * True energy axis
-* ``THETA_LO``, ``THETA_HI`` -- ndim: 1
+    * Reconstructed energy axis
+* ``THETA_LO``, ``THETA_HI`` -- ndim: 1, unit: deg
     * Field of view offset axis
+    * Binning is often chosen with a square root scale,
```

Comment:
I would suggest you remove this comment on binning choice here.

Such a comment could go in a general notes section somewhere on IRF production, but it doesn't really have anything to do with this HDU specifically, and also, I think the current statement is just wrong and we shouldn't argue about it here in this PR (equal solid angle would mean  larger `THETA_HI - THETA_LO` in the center of the FOV, you say smaller; and also I would argue that the binning should be chosen with a square scale, not a sqrt scale, i.e. the other way around, to capture the shape of the IRF dependency with as few nodes as possible for efficient storage. This is what https://fermi.gsfc.nasa.gov/ssc/data/analysis/scitools/help/gtpsf.txt does e.g.; of course that would mean for the FOV offset dependency that one shouldn't use diffuse MC to produce the IRF -- you see: a huge controversial topic to be discussed some other time, not in this spec for `RAD_MAX` theta binning.